build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
     compile 'com.github.kittinunf.fuel:fuel-rxjava:1.9.0'
     compile group: 'org.eclipse.jgit', name: 'org.eclipse.jgit',
             version: '4.8.0.201706111038-r'
+    compile "org.slf4j:slf4j-nop:1.7.2"
```

Comment:
not sure how this exactly works, I trust you though on this one.

Reply:
JGit use SLF4J API, nop is an empty implementation of this api (stands for no operation).